frontend/pulse3d/src/pages/uploads.js

```diff
@@ -156,6 +144,46 @@ export default function Uploads() {
   const [filtercolumn, setFilterColumn] = useState("");
   const [updateData, toggleUpdateData] = useState(false);
 
+  const uploadTableColumns = [
+    {
+      name: "Select",
+      center: false,
+      admin: false,
+      selector: (row) => (
+        <Checkbox id={row.id} checked={checkedUploads.includes(row.id)} onChange={handleCheckedUploads} />
+      ),
+    },
+    {
+      name: "File Owner",
+      admin: true,
+      center: false,
```

Comment:
This value is defined in `columnProperties`, you can remove it here and all the other objs in this array